waku/v2/protocol/lightpush/waku_lightpush_test.go

```diff
@@ -264,6 +265,9 @@ func TestWakuLightPushCornerCases(t *testing.T) {
 
 	// Prepare peer manager instance to include in test
 	pm := peermanager.NewPeerManager(10, 10, nil, nil, true, utils.Logger())
+	pc, err := peermanager.NewPeerConnectionStrategy(pm, onlinechecker.NewDefaultOnlineChecker(true), 10*time.Second, utils.Logger())
```

Comment:
why is this required for a lightnode? 

peerManager alone should be sufficient right.